.github/workflows/dev_labeler/labeler.yml

```diff
@@ -16,7 +16,7 @@
 # under the License.
 
 rust-lang:
```

Comment:
We use `lang-rust` not `rust-lang`: https://github.com/apache/arrow/pulls?q=is%3Apr+is%3Aopen+label%3Alang-rust

Should we rename this to `lang-rust`?

Reply:
I am now ready to retire to a temple and never touch a computer again 😞

Thanks a lot for saving this, @kou . 💯